content/how-to/ignore-rules/local/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Configuring local ignore rules"
```

Comment:
Long titles are good for FAQs, but shorter titles work better in the usage guide. I wouldn't have changed this one.